docs/source/llms/transformers/tutorials/conversational/pyfunc-chat-model.ipynb

```diff
@@ -0,0 +1,437 @@
+      "env: TOKENIZERS_PARALLELISM=false\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Disable tokenizers warnings when constructing pipelines\n",
+    "%env TOKENIZERS_PARALLELISM=false\n",
+    "\n",
+    "import warnings\n",
+    "\n",
+    "# Disable a few less-than-useful UserWarnings from setuptools and pydantic\n",
+    "warnings.filterwarnings(\"ignore\", category=UserWarning)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Serving Chat Model with Transformers\n",
+    "\n",
+    "MLflow's native Transformers integration allows you to specify the `task` param when saving or logging your `text-generation` pipelines. This param can be one of two string literals, conforming to the [MLflow Deployments Server's endpoint_type specification](https://mlflow.org/docs/latest/llms/deployments/index.html#general-configuration-parameters) (\"llm/v1/embeddings\" can be specified as a task on models saved with `mlflow.sentence_transformers`):\n",
+    "\n",
+    "- \"llm/v1/chat\" for chat-style applications\n",
+    "- \"llm/v1/completions\" for generic completions\n",
+    "\n",
+    "When the `task` param is specified, MLflow will automatically handle everything required to serve a chat model. This includes:\n",
+    "\n",
+    "- Setting a chat-compatible signature on the model\n",
+    "- Performing data pre- and post-processing to ensure the inputs and outputs conform to the [Chat API spec](https://mlflow.org/docs/latest/llms/deployments/index.html#chat), which is compatible with OpenAI's API spec.\n",
+    "\n",
+    "Note that these modifications only apply when the model is loaded with `mlflow.pyfunc.load_model()` (e.g. when serving the model with the `mlflow models serve` CLI tool). If you want to load just the base pipeline, you can always do so via `mlflow.transformers.load_model()`.\n",
+    "\n",
+    "Here's an example of how to save a Transformers model with the `task` param using TinyLlama-1.1B-Chat:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "/var/folders/qd/9rwd0_gd0qs65g4sdqlm51hr0000gp/T/ipykernel_74928/2826724314.py:10: FutureWarning: The 'transformers' MLflow Models integration is known to be compatible with the following package version ranges: ``4.25.1`` -  ``4.37.1``. MLflow Models integrations with transformers may not succeed when used with package versions outside of this range.\n",
+      "  mlflow.transformers.save_model(\n"
+     ]
+    }
+   ],
+   "source": [
+    "from transformers import pipeline\n",
+    "\n",
+    "import mlflow\n",
+    "\n",
+    "generator = pipeline(\n",
+    "    \"text-generation\",\n",
+    "    model=\"TinyLlama/TinyLlama-1.1B-Chat-v1.0\",\n",
+    ")\n",
+    "\n",
+    "mlflow.transformers.save_model(\n",
+    "    transformers_model=generator, path=\"tinyllama-chat\", task=\"llm/v1/chat\"\n",
```

Comment:
[Not in the scope of this PR] Just realized that now an input example has to be string i.e. the one for original pipeline, not ChatModel. Created a JIRA in the next sprint:)